README.md

```diff
@@ -132,6 +132,10 @@ Debian, Ubuntu, and related distributions will also need a copy of the developme
 
 If using an older release or a long-term support release, be careful to double-check that the version of `runc` is new enough (running `runc --version` should produce `spec: 1.0.0`), or else build your own.
 
+**NOTE**
+
+Be carefule to double-check that the version of golang is new enough, version 1.8.x or higher is required.  Please verify your golang version with `go version`.  golang kits are avaliable at https://golang.org/dl/ 
```

Comment:
`s/carefule/careful`

Reply:
also drop the `golang kits are...` and `Please verify your golang version with ...` - no need to specify these

Reply:
I'm hip with dropping the version check if you like, but I really like having a pointer to a kit to fix my issue.  Drives me bonkers if I have to go googling for it when I could have just done one click.  Let me retry once more.